### ISSUE TITLE
Splash Screen does not resize(Scaling Issue)

### ISSUE THREAD

**Issue by nssidhu** [dotnet/maui#4341]
### Description

Based on this article https://docs.microsoft.com/en-us/dotnet/maui/user-interface/images/splashscreen?tabs=android,
I was expecting that splash screen would resize, but it does not.

Here is the repo of the project
[Source Code in Github](https://github.com/nssidhu/AppIconAndSplashScreenSample)

Also here is the screen recording for the issue.
[Screen Recording](https://1drv.ms/v/s!AtHk7h6Srnk_h-BsbjyTex-LSTZP2w?e=Hf4CPf)

### Steps to Reproduce

I am using VS 2002  17.1.0 preview 4.0
1. Create New .NET MAUI Project
2. Change the Splash Screen to new .png file
3.Run the App, Splash screen does display but in actual size(oversized for the mobile screen), as per the article was expecting it to appropriately size for the platform

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android

### Affected platform versions

iOS15 Android 11

### Did you find any workaround?

NO

**Comment by kristinx0211** [comment]
verified on Android 11, repro sample:
[#4341.zip](https://github.com/dotnet/maui/files/8168515/4341.zip)

**Comment by mattleibow** [comment]
I just need to clarify what you mean by:

> I was expecting that splash screen would resize, but it does not.

I see you have 2 images, one small and one large. From what I see is that the image is the original size and you expected it to resize in some way?

What are you thinking should happen? Big sizes shrink? Small sizes grow? Both? Something else?

**Comment by nssidhu** [comment]
I am talking about Larger image, it should have resized to smaller size and be centered.
==========================================================================================================
    On Tuesday, March 22, 2022, 04:48:03 PM CDT, Matthew Leibowitz ***@***.***> wrote:  
 
 


I just need to clarify what you mean by:


I was expecting that splash screen would resize, but it does not.


I see you have 2 images, one small and one large. From what I see is that the image is the original size and you expected it to resize in some way?

What are you thinking should happen? Big sizes shrink? Small sizes grow? Both? Something else?

**Comment by maxkoshevoi** [comment]
I have the same issue. Image specified in `<MauiSplashScreen>` gets displayed full screen, instead of taking just a portion of it. Autoscaling would be great, but being able to specify image size in `<MauiSplashScreen>` would also solve the issue

**Comment by angelru** [comment]
any progress on this?

**Comment by Cywizz** [comment]
I also followed the referenced article above and I cannot get the splash screen to resize properly. I have the .svg in the Resource/Splash folder, and it has build action as MAUISplashScreen
I set the base size to 1072, 1768 (Divisible by 8) and gave it a White background color.
It does not resize at all.

Also, the reference mentioned that maui will create the **Resources/values/maui_colors.xml** and the **Resources/drawable/maui_splash_image.xml** files for Android, which it did not... I assume it is one of the issues here?

**Comment by J-Swift** [comment]
I'm getting a build error talking about the same missing assets @Cywizz mentions:

```
obj/Debug/net6.0-android/resizetizer/sp/maui_splash_image.xml : error XA2001: Source file 'obj/Debug/net6.0-android/resizetizer/sp/maui_splash_image.xml' could not be found. [...]
obj/Debug/net6.0-android/resizetizer/sp/maui_colors.xml : error XA2001: Source file 'obj/Debug/net6.0-android/resizetizer/sp/maui_colors.xml' could not be found. [...]
```

I have the following in my csproj (and its been there since before upgrading Maui):

```
<MauiSplashScreen Include="Resources\appiconfg.svg" Color="#512BD4" />
```

**Comment by mattleibow** [comment]
I think this is a "duplicate" of #9797 and will be fixed with #9794
